insights/parsers/ifcfg.py

```diff
@@ -79,6 +79,7 @@ class IfCFG(LegacyItemAccess, CommandParser):
 
     def __init__(self, context):
         super(IfCFG, self).__init__(context)
+        self.context = context
```

Comment:
Better change this to
```self._has_empty_line = any(l.strip() == '' for l in context)```

Reply:
@xiangce and remove the `@property` ?

Reply:
The test will fail

```bash
>       assert r._has_empty_line is False
E       AttributeError: 'IfCFG' object has no attribute '_has_empty_line'

```

Reply:
I don't think it will fail after you added the assignment e.g. `self._has_empty_line =` to the `__init__`

Reply:
Done. Please check